src/xdocs/config_imports.xml

```diff
@@ -997,6 +997,64 @@ public class InputIllegalImport { }
         </source>
 
         <p>
+          In the next examples usage of <code>strategyOnMismatch</code> property is shown.
+          This property defines strategy in a case when no matching allow/disallow rule was found.
+          Property <code>strategyOnMismatch</code> is attribute of <code>import-control</code> and <code>subpackage</code> tags.
+          Property can have following values for <code>import-control</code> tag:
+          <ul>
+            <li>disallowed (default value) - if there is no matching allow/disallow rule in any of the subpackages, including the root level (import-control), then the import is allowed.</li>
+            <li>allowed - if there is no matching allow/disallow rule in any of the subpackages, including the root level, then the import is disallowed.</li>
```

Comment:
Don't you have the ending swapped for each value?

> disallowed ... then the import is allowed.
> allowed ... then the import is disallowed

Shouldn't `disallowed` say "then the import is disallowed"?
The next section doesn't have this issue.